### PR TITLE
first pass on 07-testing

### PR DESCRIPTION
**Pull request by radifar**
Added new lines in every end of sentence. Fix few typos.

### CODE REVIEW

File: _episodes/07-testing.md [hunk 1]
```diff
@@ -140,11 +158,13 @@ molecool/tests/test_molecool.py::test_molecool_imported PASSED [100%]
...
 
-Now we see that `pytest` dsiplays the test name for us, as well as `PASSED` next to the test name.
+Now we see that `pytest` displays the test name for us, as well as `PASSED` next to the test name.
 
 ## Testing our functions
 
-We will now add tests to test our functions. After dividing our package into modules, we have four modules and one subpackage. It is considered good practices to also break your tests into different files depending on the module or subpackage.
+We will now add tests to test our functions.
+After dividing our package into modules, we have four modules and one subpackage.
+It is considered good practices to also break your tests into different files depending on the module or subpackage.
```
Comment (eirrgang) [inline _episodes/07-testing.md:167]:
```suggestion
It is considered good practice to also break your tests into different files depending on the module or subpackage.
```

File: _episodes/07-testing.md [hunk 2]
```diff
@@ -591,7 +645,10 @@ molecool/tests/test_measure.py::test_calculate_distance SKIPPED
 
 You might also use the `pytest.mark.xfail` if you expect a test to fail.
 
-You can also define your own marks. Let's consider if some of our tests were slow or took a long time. Maybe we would not want to run these tests every time. We could add our own mark
+You can also define your own marks.
+Let's consider if some of our tests were slow or took a long time.
+Maybe we would not want to run these tests every time.
+We could add our own mark
```
Comment (eirrgang) [inline _episodes/07-testing.md:651]:
```suggestion
We could add our own mark:
```

**Review by eirrgang — CHANGES_REQUESTED** [review]

**Comment by eirrgang** [comment]
Relates to https://github.com/MolSSI-Education/python-package-best-practices/pull/13

File: _episodes/07-testing.md [hunk 3]
```diff
@@ -10,47 +10,52 @@ objectives:
...
-- **System tests**: where you test your system as a whole to check if it meets all the
-requirements.
+- **System tests**:
+Where you test your system as a whole to check if it meets all the requirements.
 
-Another important type of testing is **Regression tests**. In Regression tests,
-given a known input, does the software correctly and consistently return the correct
-values? This kind of testing can catch problems in previously working code that may has been broken by new changes or new features.
+Another important type of testing is **Regression tests**.
+In Regression tests, given a known input, does the software correctly and consistently return the correct values?
```
Comment (janash) [inline _episodes/07-testing.md:54]:
```suggestion
In Regression tests, given a known input, does the software correctly and consistently return the correct values.
```

**Review by janash — COMMENTED** [review]
Looks good as a first pass. Is it ready to merge?

File: _episodes/07-testing.md (hunk 3 above)
Comment (radifar) [reply _episodes/07-testing.md:54]: Sorry, missed this one.

**Comment by radifar** [comment]
OK, ready to merge.

**Review by janash — APPROVED** [review]

**Pull request merged**